docs/docs/guide/02-getting-started.md

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_position: 2
+---
+
+# Getting started
+
+In this tutorial, we will be using Ignite CLI to create a new blockchain. Ignite
+CLI is a command line interface that allows users to quickly and easily create
+blockchain networks. By using Ignite CLI, we can quickly create a new blockchain
+without having to manually set up all the necessary components.
+
+Once we have created our blockchain with Ignite CLI, we will take a look at the
+directory structure and files that were created. This will give us an
+understanding of how the blockchain is organized and how the different
+components of the blockchain interact with each other.
+
+By the end of this tutorial, you will have a basic understanding of how to use
+Ignite CLI to create a new blockchain, and you will have a high-level
+understanding of the directory structure and files that make up a blockchain.
+This knowledge will be useful as you continue to explore the world of blockchain
+development.
+
+## Creating a new blockchain
+
+To create a new blockchain project with Ignite, you will need to run the
+following command:
+
+```
+ignite scaffold chain example
+```
+
+The [`ignite scaffold chain`](/cli#ignite-scaffold-chain) command will create a
+new blockchain in a new directory `example`.
+
+The new blockchain is built using the Cosmos SDK framework and imports several
+standard modules to provide a range of functionality. These modules include
+`staking`, which enables a delegated Proof-of-Stake consensus mechanism, `bank`
+for facilitating fungible token transfers between accounts, and `gov` for
+on-chain governance. In addition to these modules, the blockchain also imports
+other modules from the Cosmos SDK framework.
```

Comment:
What about mentioning the `example` module ? 

More broadly, I found that it's typically something that is not really well explained to new comers (in ignite or even in the cosmos-sdk docs) : the fact that the modules determine the *features* of the blockchain.

Once we said that, we can mention that `ignite scaffold chain` has automatically created a custom module named `example` where the specific feature of the blockchain should be implemented.

Maybe it's just me but that's something that confused me at the very beginning :)

Reply:
> the fact that the modules determine the _features_ of the blockchain.

OK you define that below at the `/x` section :pray:.

Reply:
https://github.com/ignite/cli/pull/3218/commits/896a17f5f20fd40f73091573f61858b6009ec5ac